corehq/apps/geospatial/tasks.py

```diff
@@ -1,7 +1,13 @@
 from corehq.apps.celery import task
-from corehq.apps.geospatial.utils import update_cases_owner
+from corehq.apps.geospatial.utils import CeleryTaskExistenceHelper, update_cases_owner
+
+
+@task(queue="background_queue", ignore_result=True)
```

Comment:
`ignore_result=True` does not store the state of the celery task in the results back end. (HQ uses django for results backend). 
This was added since we no longer rely on the results backend and instead use redis.

Reply:
Good to know, thank you!